lms/views/dashboard.py

```diff
@@ -39,5 +39,18 @@ def assignment_show(self):
         renderer="json",
         permission=Permissions.DASHBOARD_VIEW,
     )
-    def api_assignment_analytics(self):  # pragma: nocover
-        return {}
+    def api_assignment_stats(self):
+        assignment = self.assignment_service.get_by_id(self.request.matchdict["id_"])
+        stats = self.h_api.get_assignment_stats(
```

Comment:
We should filter out teacher here and include students not returned by the H api (because they don't have any annotations).

Added issues to the board. Focusing now on getting things working end to end.